components/OwlCarousel.jsx

```diff
@@ -208,7 +207,7 @@ class OwlCarousel extends React.Component {
 		} = this.props;
 
 		return (
-			<div className='owl-carousel owl-theme' {...props}>
+			<div ref={(item) => this.$node = $(findDOMNode(item))} className='owl-carousel owl-theme' {...props}>
```

Comment:
Hydrating a react serverside app requires this to be the way you reference the node. (Not 100% sure why the previous code does not work as expected, but it does not.)